modules/claims_api/config/routes.rb

```diff
@@ -39,6 +39,7 @@
       get '/:veteranId/claims', to: 'claims#index'
       get '/:veteranId/claims/:id', to: 'claims#show'
       get '/:veteranId/power-of-attorney', to: 'power_of_attorney#show'
+      post '/:veteranId/claims/:id/5103/', to: 'evidence_waiver#submit'
```

Comment:
```suggestion
      post '/:veteranId/claims/:id/5103/', to: 'evidence_waiver#submit'
      get '/:veteranId/power-of-attorney', to: 'power_of_attorney#show'
```

I was thinking one more up so group them with the other `/claims`